cli/package.json

```diff
@@ -175,5 +175,15 @@
     "nohoist": [
       "@types/*"
     ]
+  },
+  "nx": {
+    "targets": {
+      "build": {
+        "outputs": [
+          "{projectRoot}/types",
```

Comment:
We have to specify these outputs because we generate `/types` in CI. By default, Nx looks in [the following directories](https://nx.dev/reference/project-configuration#outputs) so deviations from those standards need to be specified.